algebird-core/src/main/scala/com/twitter/algebird/Aggregator.scala

```diff
@@ -103,6 +103,16 @@ object Aggregator extends java.io.Serializable {
   def sortedReverseTake[T: Ordering](count: Int): MonoidAggregator[T, PriorityQueue[T], Seq[T]] =
     new mutable.PriorityQueueToListAggregator[T](count)(implicitly[Ordering[T]].reverse)
   /**
+   * Immutable version of sortedTake
```

Comment:
I've seen frameworks that check immutability of reduce functions, for example [https://github.com/GoogleCloudPlatform/DataflowJavaSDK/commit/ca2d22e198212c38e76b8b91ae8698dc2d877666](Dataflow). This would work around that. Also for the sake of completeness?
It's not very intuitive to write something like:

```scala
Aggregator
    .prepareMonoid[T, TopK[T]](i => new TopK(1, List(i), Some(i)))(new TopKMonoid[T](max)(ord))
    .andThenPresent(_.items.toSeq)
```